examples/assembler/cube/data/test.vsh

```diff
@@ -32,8 +32,8 @@ proj:  // result.pos = projMtx * tempreg
 
 tex:  // result.texcoord = in.texcoord
 	mov outtex0,  v1
-	mov outtex1,  c20.yyyw
-	mov outtex2,  c20.yyyw
+	mov outtex1,  c32.yyyw
```

Comment:
Would it also make sense to use "outpos.x" instead of "o0.x" in the above lines for proj matrix?